src/System.Diagnostics.Process/ref/System.Diagnostics.Process.Manual.cs

```diff
@@ -10,8 +10,18 @@ namespace System.Diagnostics
     public partial class Process : System.IDisposable
     {
         public void Dispose() { }
+        /// <summary>
+        /// Release all resources used by this process.
+        /// </summary>
+        /// <param name="disposing">
+        /// true to release both managed and unmanaged resources; false to release only unmanaged resources.
```

Comment:
What's the aim here? To generate docs off these things or just intellisense? If both, then you're going to be losing information, including remarks, inheriting info, langwords, code samples, etc. What's the plan around this?